docs/source/tips_and_tricks.rst

```diff
@@ -62,3 +62,16 @@ One of the main advantages of using **signac** for data management is that the s
 That also means that existing ids will change and scripts that used them in a hard-coded fashion will fail.
 
 Whenever you find yourself hard-coding ids into your code, consider replacing it with a function that uses the :py:meth:`~.signac.Project.find_jobs` function instead.
+
+
+How do I achieve optimal performance? What are the practical scaling limits for project sizes?
+----------------------------------------------------------------------------------------------
+
+Because **signac** uses a filesystem backend, there are some practical limitations for project size.
```

Comment:
```suggestion
Because **signac** uses a filesystem backend, there are some practical limitations for project size, and while there is no hard limit some heuristics can be helpful.
```